Kiss.Bff/Extern/ZaakGerichtWerken/Zaaksysteem/GetZaken.cs

```diff
@@ -120,15 +129,33 @@ public async Task<IActionResult> Get([FromRoute] string version, [FromQuery(Name
             return Ok(result);
         }
 
-        private Task<(Uri? BaseAddress, HttpResponseMessage? Message)[]> SendRequestToZaaksystemen(string path, CancellationToken token)
+        private Task<(Uri? BaseAddress, HttpResponseMessage? Message)[]> SendRequestToZaaksystemen(string path, string? kvkNummer, string? rsin, CancellationToken token)
         {
             var tasks = _configs
-                .Select(CreateClient)
-                .Select(async (client) =>
+                .Select(async (config) =>
                 {
+                    var client = CreateClient(config);
                     try
                     {
-                        var message = await client.GetAsync($"{path}{Request?.QueryString}", HttpCompletionOption.ResponseHeadersRead, token);
+                        var queryString = Request?.QueryString.ToString();
+                        
+                        if (!string.IsNullOrWhiteSpace(kvkNummer) && !string.IsNullOrWhiteSpace(rsin))
+                        {
+                            var identifier = config.NietNatuurlijkPersoonIdentifier == "kvkNummer"
```

Comment:
"kvkNummer" moet een podiumD brede constante worden. zit zo erg verstopt in de code.  zouden we bijvoorbeeld  in een constants.cs kunnen stoppen. misschien heb je een beter idee

Reply:
Niet per se podiumd breed, toch? Het gaat om 
- de waarde die je in een environment variabele stopt voor kiss
- een query parameter die van de frontend naar de backend gestuurd wordt binnen kiss

Reply:
Besproken: taakje aanmaken om de handleiding te updaten.